comp/api/api/apiimpl/api.go

```diff
@@ -61,12 +62,13 @@ type apiServer struct {
 	rcService             optional.Option[rcservice.Component]
 	rcServiceHA           optional.Option[rcserviceha.Component]
 	authToken             authtoken.Component
+	endpointProviders     []api.AgentEndpointProvider
 }
 
 type dependencies struct {
 	fx.In
 
-	Flare                 flare.Component
+	// Flare                 flare.Component
```

Comment:
we can remove the flare as a dependency in the component now since it registers it's own route